### PR TITLE
chore(codegen): bump codegen version to 0.19.0

### PR DESCRIPTION
**Pull request by hpmellema**
### Description
Bump codegen version to 0.19.0

Generated clients have a diff for exporting `RuntimeExtension` and Client `ExtensionConfiguration`, e.g. for Access Analyzer:

```diff
diff --git a/clients/client-accessanalyzer/src/index.ts b/clients/client-accessanalyzer/src/index.ts
index 50b2e0692a6..74d473da178 100644
--- a/clients/client-accessanalyzer/src/index.ts
+++ b/clients/client-accessanalyzer/src/index.ts
@@ -16,6 +16,8 @@
 export * from "./AccessAnalyzerClient";
 export * from "./AccessAnalyzer";
 export { ClientInputEndpointParameters } from "./endpoint/EndpointParameters";
+export { RuntimeExtension } from "./runtimeExtensions";
+export { AccessAnalyzerExtensionConfiguration } from "./extensionConfiguration";
 export * from "./commands";
 export * from "./pagination";
 export * from "./models";
```

Also added experimental identity and auth private client package.

---
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: scripts/generate-clients/config.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 // Update this commit when taking up new changes from smithy-typescript.
 module.exports = {
   // Use full commit hash as we explicitly fetch it.
-  SMITHY_TS_COMMIT: "93274e37a28e508e5137493ffdf7aa63b0c8daed",
+  SMITHY_TS_COMMIT: "6db52d8fd544697a80a50d54f4731aae9eec40a9",
```
Comment (syall) [inline scripts/generate-clients/config.js:4]: TODO: This should be updated to the smithy-typescript main commit of https://github.com/awslabs/smithy-typescript/pull/1060

Marking my review as requesting changes until this is done

**Review by syall — CHANGES_REQUESTED** [review]

File: lib/lib-dynamodb/src/index.ts [hunk 2]
```diff
@@ -4,5 +4,3 @@ export * from "./DynamoDBDocumentClient";
 // smithy-typescript generated code
 export * from "./commands";
 export * from "./pagination";
-
-export { NumberValueImpl as NumberValue } from "@aws-sdk/util-dynamodb";
```
Comment (syall) [inline lib/lib-dynamodb/src/index.ts:8]: Is this export being removed expected?

File: private/weather/package.json [hunk 3]
```diff
@@ -10,7 +10,7 @@
     "build:types": "tsc -p tsconfig.types.json",
     "build:types:downlevel": "downlevel-dts dist-types dist-types/ts3.4",
     "clean": "rimraf ./dist-* && rimraf *.tsbuildinfo",
-    "test": "vitest run"
+    "test": "vitest run --passWithNoTests"
```
Comment (syall) [inline private/weather/package.json:13]: This is code generated from https://github.com/awslabs/smithy-typescript/pull/1065

File: lib/lib-dynamodb/src/index.ts (hunk 2 above)
Comment (kuhe) [reply lib/lib-dynamodb/src/index.ts:8]: fixed in https://github.com/aws/aws-sdk-js-v3/pull/5456

**Review by syall — APPROVED** [review]

**Review by kuhe — APPROVED** [review]

**Pull request merged**